scalafix-sbt/src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -50,6 +56,19 @@ object ScalafixPlugin extends AutoPlugin {
     cliWrapperMainClass := "scalafix.cli.Cli$",
     scalafixEnabled := true,
     scalafixVerbose := false,
+    scalafixBuild := Def.inputTaskDyn {
```

Comment:
Currently scalafix will run and semantic rewrites will get no symbols/denotations, which may be confusing for users. Ideally, in the future, we can trigger a reload with the compiler plugin enabled to guarantee that the .semanticdb files are in sync with the sources.

I'm not sure if we can query the libraryDependencies of metabuilds from here. When invoked, the task is typically scoped to the root project. I'll add a note to explain.